opendebates/templates/flatpages/social_override.html

```diff
@@ -0,0 +1,22 @@
+{% extends "opendebates/base.html" %}
+
+{% block metadata %}
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:title" content="HAPPENING NOW! Watch history be made.">
+<meta name="twitter:description" content="Well, this is awesome. I'm watching Democratic Congressman Alan Grayson debate Republican Congressman David Jolly in Florida. It's the first time ever U.S. Senate candidates are participating in an Open Debate where all questions where submitted and voted on by the public -- and moderators must ask top-voted questions. Watch it with me...">
```

Comment:
I think the second 'where' should be 'were'? Otherwise this looks fine. @ejucovy, I'm happy to make the change and start the testing deploy. Just want your confirmation.